internal/os_updater/ubuntu.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-FileCopyrightText: (C) 2025 Intel Corporation
+ * SPDX-License-Identifier: LicenseRef-Intel
```

Comment:
INBM is `Apache-2.0`

Reply:
oh yeah, I think I copied these from UDM which is why it's incorrect.